lib/tokenpricer/coingecko/coingecko.go

```diff
@@ -72,16 +73,42 @@ func (c Client) USDPrice(ctx context.Context, tkn tokens.Asset) (float64, error)
 
 // USDPrices returns the price of each coin in USD.
 func (c Client) USDPrices(ctx context.Context, tkns ...tokens.Asset) (map[tokens.Asset]float64, error) {
+	rats, err := c.getPrice(ctx, currencyUSD, tkns...)
+	if err != nil {
+		return nil, errors.Wrap(err, "get price")
+	}
+
+	prices := make(map[tokens.Asset]float64)
+	for tkn, price := range rats {
+		f, _ := price.Float64()
+		prices[tkn] = f
+	}
+
+	return prices, nil
+}
+
+// USDPricesRat returns the price of each coin in USD.
+func (c Client) USDPricesRat(ctx context.Context, tkns ...tokens.Asset) (map[tokens.Asset]*big.Rat, error) {
 	return c.getPrice(ctx, currencyUSD, tkns...)
 }
 
+// USDPriceRat returns the price of each coin in USD.
```

Comment:
```suggestion
// USDPriceRat returns the price of a coin in USD.
```

could probably hange coin to token in places